srcs/parsing/map_to_list.c

```diff
@@ -57,8 +57,8 @@ int	ft_map_to_list(t_cub_data *data, char *map_file)
 		if (line && ft_add_line_map(line, data) == WRONG_MALLOC)
 			return (get_next_line(GNL_FLUSH), ft_err_msg("Malloc error."));
 	}
-	ft_lstiter(data->lst_map, &ft_print_lst_map);
-	ft_putstr_fd("\n", 1);
+	// ft_lstiter(data->lst_map, &ft_print_lst_map);
```

Comment:
As mentioned files merged with master should respect the norm.
Therefore these commented lines should be moved out of the scope or deleted.

Reply:
I uncommented it in the past whooooosh